parl/plutils/common.py

```diff
@@ -82,11 +78,10 @@ def get_parameter_pairs(src, target):
             target_var = getattr(target, attr)
             param_pairs.extend(get_parameter_pairs(src_var, target_var))
     elif isinstance(src, LayerFunc):
-        param_pairs.append((src.param_attr.name, target.param_attr.name,
-                            False))
-        if src.bias_attr:
-            param_pairs.append((src.bias_attr.name, target.bias_attr.name,
-                                True))
+        for (src_attr, target_attr) in zip(src.attr_holder.sorted(),
```

Comment:
How can you guarantee that `src.attr_holder` and `target.target_holder` have same number of parameters.

Reply:
added assertion here.